onnxutils/onnxconverter_common/utils.py

```diff
@@ -196,7 +196,7 @@ def is_string_type(item):
     return isinstance(item, types)
 
 
-def _check_has_attr(obj, attribute):
+def check_has_attr(obj, attribute):
```

Comment:
well, it's not a good practice to rename a python function, which looks weird and unncessary.
Let's remove it.

Reply:
Got it! I'll replace it's use in the SVM converter with the native Python has_attr.